xyz.eclipium.TetraSwap.json

```
@@ -0,0 +1,64 @@
+{
+    "app-id": "xyz.eclipium.TetraSwap",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "TetraSwap",
+	"rename-appdata-file": "TetraSwap.appdata.xml",
+	"rename-icon": "TetraSwap",
+	"rename-desktop-file": "TetraSwap.desktop",
+    "finish-args": [
+    "--share=ipc",
+    "--socket=wayland",
+    "--socket=fallback-x11",
+    "--socket=pulseaudio",
+    "--device=dri",
+    "--share=network"
+  	],
+    "modules": [
+        {
+            "name": "TetraSwap",
+            "buildsystem": "simple",
+            "build-commands": [
+            	"ls",
+            	"mkdir /app/bin",
+				"cp TetraSwap /app/bin/",
+				"cp -r TetraSwap_Data /app/bin/",
+				"cp UnityPlayer.so /app/bin/",
+				"mkdir /app/share",
+				"mkdir /app/share/applications",
+				"cp TetraSwap.desktop /app/share/applications/",
+				"mkdir /app/share/icons",
+				"mkdir /app/share/icons/hicolor",
+				"mkdir /app/share/icons/hicolor/512x512",
+				"mkdir /app/share/icons/hicolor/128x128",
+				"mkdir /app/share/icons/hicolor/64x64",
+				"mkdir /app/share/icons/hicolor/128x128/apps",
+				"mkdir /app/share/icons/hicolor/512x512/apps",
+				"mkdir /app/share/icons/hicolor/64x64/apps",
+				"cp TetraSwap.png /app/share/icons/hicolor/512x512/apps/",
+				"cp 128.png /app/share/icons/hicolor/128x128/apps/TetraSwap.png",
+				"cp 64.png /app/share/icons/hicolor/64x64/apps/TetraSwap.png",
+				"mkdir /app/share/metainfo",
+				"cp TetraSwap.appdata.xml /app/share/metainfo/"
```

```suggestion
				"install TetraSwap.appdata.xml /app/share/metainfo/${FLATPAK_ID}.metainfo.xml"
```

And remove the `rename-appdata-file`